genesyscloud/tfexporter/resource_genesyscloud_tf_export.go

```diff
@@ -98,12 +99,18 @@ func ResourceTfExport() *schema.Resource {
 				Default:     false,
 				ForceNew:    true,
 			},
-			"export_as_hcl": {
```

Comment:
Removing this attribute is a major schema change. We should consider deprecating this attribute in favor of export_format. If export_format is not set, but export_as_hcl is, we should determine the value to use.